brainscore_language/benchmarks/syntaxgym/README.md

```diff
@@ -0,0 +1,72 @@
+SyntaxGym is a unified platform for targeted syntactic evaluation of language models. It supports all steps
+of the evaluation process, from designing test suites to visualizing final results. The goal of SyntaxGym 
+is to make psycholinguistic assessment of language models more standardized, reproducible, and accessible 
+to a wide variety of researchers.  See the [SyntaxGym Website](https://syntaxgym.org/) for more details.
+
+## YAML section for Benchmark Card
+---
+benchmark details:
```

Comment:
```suggestion
```yaml
benchmark details:
```

Reply:
This (with suggested closed quotes around line 71) should fix the syntax highlighting.

Reply:
this did indeed fix it! thanks!!